### PR TITLE
Update the size of QLabel after setting text. 

### PR DESCRIPTION
**Pull request by jaynabonne**
QLabel doesn't automatically update its size when new text is set, but wxStaticText assumes the size has been updated. So we explicitly do it.

### CODE REVIEW

File: src/qt/stattext.cpp [hunk 1]
```diff
@@ -61,6 +61,7 @@ bool wxStaticText::Create(wxWindow *parent,
 void wxStaticText::SetLabel(const wxString& label)
 {
     m_qtLabel->setText( wxQtConvertString( label ) );
+    m_qtLabel->adjustSize();
```
Comment (vadz) [inline src/qt/stattext.cpp]: You almost certainly should call `AutoResizeIfNecessary()` from here instead. If this doesn't work (which would be bad...), you need to at least check for `wxST_NO_AUTORESIZE` and not change the size if it's set (and we really ought to have a unit test for this not quite obvious behaviour...).

**Review by vadz — CHANGES_REQUESTED** [review]
Sorry, I don't think this is quite right, as per the comment below.

File: src/qt/stattext.cpp (hunk 1 above)
Comment (jaynabonne) [reply src/qt/stattext.cpp]: That makes good sense. I have made the change, and it works fine. Thanks!

**Review by catalinr — COMMENTED** [review]
The requested change has been applied, so I guess this can be merged.

**Comment by vadz** [comment]
I think I've fixed this in the meanwhile in 958521183a2547123dfe7fb53e9a726002b08593, please let me know if there are still any problems with this.

**Pull request closed without merging**